api/models/v0.3/modelCard.js

```diff
@@ -166,6 +175,7 @@ class Card {
     this.address = this.getOrCreateAddress(args["address"]);
     this.workAddress = this.getOrCreateAddress(args["workAddress"]);
     this.username = args["username"];
+    this.usernameHasBeenValidated = args["usernameHasBeenValidated"] || false;
```

Comment:
Again, if it's just a boolean, `!!args["usernameHasBeenValidated"]` would be more readable...